default.nix

```diff
@@ -49,22 +49,6 @@ let haskellPackages =
                         mkPackage super "streamly-examples"
                             ./. flags inShell;
 
-                    streamly =
```

Comment:
Why remove this? We will need to add this again for 0.8.1 when we release.

Reply:
We'll add this back when we release streamly.